File: 프로그래머스/lv1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
    let answer = [];
    arr.forEach(element => {
        if (element % divisor == 0){
            answer.push(element);
        }
    });
    
    if (answer.length == 0) answer.push(-1);
    else {
        answer.sort((a,b) => (a-b));
    }
    return answer; 
}

File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        
        List<Integer> list = new ArrayList<>();
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] % divisor == 0) {
                list.add(arr[i]);
            }
        }
        
        if (list.size() == 0) list.add(-1);
        
        int[] nums = list.stream().mapToInt(i -> i).toArray();
        Arrays.sort(nums);
        
        return nums;
    }
}